### PR TITLE
[Merged by Bors] - Doc fixes

### PR DESCRIPTION
**Pull request by pawanjay176**
## Issue Addressed

N/A

## Proposed Changes

Minor doc fixes. Adds a section on custom data directories.

### CODE REVIEW

File: book/src/testnet-validator.md [hunk 1]
```diff
@@ -160,3 +163,20 @@ If you see any `ERRO` (error) logs, please reach out on
 issue](https://github.com/sigp/lighthouse/issues/new).
 
 Happy staking!
+
+
+## Custom directories
+
+Users can override the default Lighthouse data directories (`~/.lighthouse/{testnet}`) using the `--datadir` flag. The custom data directory mirrors the structure of any testnet specific default directory (e.g. `~/.lighthouse/medalla`).
+
+> Note: Users should specify different custom directories for different testnets.
```
Comment (realbigsean) [inline book/src/testnet-validator.md:172]: I think it would be good to add this note, or something similar to the help section of the  `--datadir` flag

**Review by realbigsean — APPROVED** [review]
Looks good! Just had one suggestion but approved otherwise

**Review by realbigsean — APPROVED** [review]

File: book/src/key-recovery.md [hunk 2]
```diff
@@ -43,10 +43,12 @@ keystores that correspond to one or more indices in the mnemonic:
 
 
 For each of the indices recovered in the above commands, a directory will be
-created in the `--validator-dir` location (default `~/.lighthouse/validator`)
+created in the `--validator-dir` location (default `~/.lighthouse/{testnet}/validator`)
```
Comment (michaelsproul) [inline book/src/key-recovery.md:46]:
```suggestion
created in the `--validator-dir` location (default `~/.lighthouse/{testnet}/validators`)
```

File: book/src/validator-import-launchpad.md [hunk 3]
```diff
@@ -25,7 +25,7 @@ website). If this is not the case, simply change the `--directory` to point to
 the `validator_keys` directory.
 
 Now, assuming that the user is in the `eth2-deposit-cli` directory and they're
-using the standard `validators` directory (specify a different one using
+using the default (`~/.lighthouse/{testnet}/validators`) `validators` directory (specify a different one using
 `--validator-dir` flag), they can follow these steps:
```
Comment (michaelsproul) [inline book/src/validator-import-launchpad.md:29]:
```suggestion
`--validators-dir` flag), they can follow these steps:
```

**Review by michaelsproul — APPROVED** [review]
Approved modulo the typos I fixed

**Pull request closed without merging**